express-get-json/index.js

```diff
@@ -0,0 +1,40 @@
+const express = require('express');
+const app = express();
+
+app.listen(3000, () => {
+  // eslint-disable-next-line no-console
+  console.log('Express server is listening!');
+});
+
+var grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+const gradesArr = [];
+
+for (const key in grades) {
+
+  gradesArr.push(grades[key]);
+
+  app.get('/api/grades', (req, res) => {
+    res.json(gradesArr);
+  });
+
+}
```

Comment:
When making multiple requests, each request should result in a response containing the same amount of data.